tests/test_utils.py

```diff
@@ -18,6 +18,11 @@ def setUp(self):
         self._example_for_spacy_tokenization = "Company A has a profit margin of 10% and B has 20%."
         self._words_and_phrases = ["batch", "coconut", "actor", "wheel", "blood pressure medicine"]
         self._spacy_sentence = nlp('Let us learn about " puff of air test " and a sewing - machine')
+        self._unnormalized_text = "  This\nis\r\na\nstring\n\rwith\n\rmultiple\n\r\nnewlines\n\n\rand   spaces.  "
```

Comment:
do you mean raw_text?

Reply:
what happens for empty "" text? Do we have a unit test for that?

Reply:
We have an empty string test here https://github.com/skadio/ner4opt/blob/66820b051bf89087f82602c2d2b004b944ddd5eb/tests/test_ner4opt.py#L78